torch_xla/csrc/cross_replica_reduces.cpp

```diff
@@ -309,6 +348,16 @@ ReduceScatterResult BuildReduceScatter(
   return {reduce_result, token_handler.GetNewToken(reduce_result)};
 }
 
+// moved from torch_xla/csrc/ops/all_reduce.cpp
```

Comment:
let's remove this comment and rename this to `GetOperandListWithToken`

Reply:
fixed